package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-rx",
-  "version": "2.1.3",
+  "version": "3.0.0-crx-749.1",
   "description": "React + RxJS = <3",
   "keywords": [
```

Comment:
Can probably remove `use-sync-external-store` from the keywords here

Reply:
Done!